lib/src/main/java/uk/gov/di/ipv/core/library/domain/UserStates.java

```diff
@@ -2,9 +2,10 @@
 
 public enum UserStates {
     DEBUG_PAGE("page-ipv-debug"),
-    INITIAL_IPV_JOURNEY("page-ipv-start"),
```

Comment:
Does this mean we should remove these templates from core-front now?

Reply:
I don't think we've used it yet, so yeah might not be needed on core-front either

Reply:
I don't think we've used it yet, so yeah might not be needed on core-front either